.gitattributes

```
@@ -0,0 +1,3 @@
+* text eol=lf
+*.gif -text
+*.psd -text
```

I think we could simplify this a bit:
```suggestion
* text=auto eol=lf
```